src/ClientResolver.php

```diff
@@ -533,10 +534,10 @@ public static function _apply_disable_request_compression($value, array &$args)
             $value = $value();
         }
         if (!is_bool($value)) {
-           throw new IAE(
-              "Invalid configuration value provided for 'disable_request_compression'."
-              . " value must be a bool."
-           );
+            throw new IAE(
```

Comment:
Hard to see this on GH, but looks like this spacing should be undone

Reply:
This is actually right.  Just verified locally.